packages/common/src/sites/_internal/capabilities/types.ts

```diff
@@ -0,0 +1,16 @@
+import { SitePermissions } from "../SiteBusinessRules";
+
+export type LegacyCapability = "hideFollow";
```

Comment:
Add doc comments to all this stuff - unless marked with `// @private` these will get into the docs, which I think is good even if the functions that use the interfaces are not exported